### ISSUE TITLE
Option to Hold button to show instead of just toggle

### ISSUE THREAD

**Issue by Aceship** [feldim2425/MoreOverlays#42]
i want to show the overlay only when holding a button  , and hide again if i unpress it.

**Comment by feldim2425** [comment]
Yes it is possible. But it would require some rewrite of my current system and I don't know how the user would be able to change the mode for the chunk borders.
